tests/smartcontract_execution_test.go

```diff
@@ -406,9 +406,10 @@ func executeSmartContractForStorageTrie(b *testing.B, opt *ContractExecutionOpti
 			fmt.Printf("run %v tx generated \n", run)
 
 			state, _ := bcdata.bc.State()
-			currentBlockNumber, rules := bcdata.bc.CurrentBlock().NumberU64(), bcdata.bc.Config().Rules(bcdata.bc.CurrentBlock().Number())
 			for _, tx := range transactions {
-				tx.AsMessageWithAccountKeyPicker(signer, state, currentBlockNumber, rules)
+				if _, err := tx.AsMessageWithAccountKeyPicker(signer, state, bcdata.bc.CurrentBlock().NumberU64()); err != nil {
+					b.Fatal(err)
```

Comment:
@yoomee1313 Why `Fatal` is added? Is this logic relavant to fork package introduction?

Reply:
> Why Fatal is added?

I added `Fatal` because other parts of tests execute `Fatal` when err is returned. 

>  Is this logic relavant to fork package introduction?

The code is not directly related to the fork package introduction. Instead, it is related to the reverse of  intrinsicGas parameter pass. 

I revise this code because when I added `SetHardForkBlockNumberConfig` logic to the `New` function at node.cn.backend.go, it returned error since it did not execute `New` function. However, it couldn't recognize the error. I just want to make sure that it catches the error.

Reply:
Thanks for explanation